src/node/desktop/src/ui/widgets/choose-r/preload.ts

```diff
@@ -132,3 +169,9 @@ const callbacks: Callbacks = {
 };
 
 contextBridge.exposeInMainWorld('callbacks', callbacks);
+
+function isRVersionSelected(selectedVersion: string, versionToCompare: string) {
+  return (
+    selectedVersion.replace(/\//g, '/').replace(/\\/g, '/') === versionToCompare.replace(/\//g, '/').replace(/\\/g, '/')
```

Comment:
Create and use utility function instead of repeating these regexprs over and over.

Reply:
Done!